webdriver/tests/bidi/script/call_function/arguments.py

```diff
@@ -17,15 +15,14 @@ async def test_arguments(bidi_session, top_context):
         await_promise=False,
         target=ContextTarget(top_context["context"]))
 
-    recursive_compare({
+    assert {
```

Comment:
Unlike in the Mapper, `recursive_compare` in WPT asserts not `==`,  but `<=` instead.

Reply:
Done